### PR TITLE
Norwegian height resource files

### PR DESCRIPTION
**Pull request by himsve**
Associated with https://github.com/OSGeo/PROJ/pull/2359

### CODE REVIEW

**Comment by rouault** [comment]
You need to edit more files. See https://github.com/OSGeo/PROJ-data/pull/17/files as an example of what is needed when adding a new authority.
We require also that the TIFF files themselves are prefix with no_kv_
Other details in https://github.com/OSGeo/PROJ-data/blob/master/CONTRIBUTING.md

**Comment by rouault** [comment]
See log at https://travis-ci.com/github/OSGeo/PROJ-data/builds/186454831 for issues to correct in the TIFF file(s)

File: CMakeLists.txt [hunk 1]
```diff
@@ -37,6 +37,7 @@ set(CPACK_SOURCE_INSTALLED_DIRECTORIES
     "${CMAKE_CURRENT_SOURCE_DIR}/is_lmi/" "."
     "${CMAKE_CURRENT_SOURCE_DIR}/nc_dittt/" "."
     "${CMAKE_CURRENT_SOURCE_DIR}/nl_nsgi/" "."
+	"${CMAKE_CURRENT_SOURCE_DIR}/no_kv/" "."
```
Comment (rouault) [inline CMakeLists.txt:40]: tag -> 4 spaces
```suggestion
    "${CMAKE_CURRENT_SOURCE_DIR}/no_kv/" "."
```

**Comment by rouault** [comment]
Why did you close this PR ?

**Comment by himsve** [comment]
Oh. I did not ment to close it. I was reverting me previous commits and did the work once more. :-)

**Pull request closed without merging**